float64[] joint_positions
float64[] actuator_pose
---
float64[] target_positions